bsvcf2bed

```diff
@@ -5,13 +5,14 @@ set -o errexit
 set -o nounset
 
 #arg1 = /path/to/input/biscuit_pileup.vcf 
-#arg2 = /path/to/output/cpgs.bed.gz
-#arg3 = /path/to/output/cpgs.bedgraph
+#arg2 = /path/to/reference.fa
+#arg3 = /path/to/output/cpgs.bed.gz
+#arg4 = /path/to/output/cpgs.bedgraph
 
 #Creates a gzipped bed and a bedgraph that leaves out MT, random, GL contigs, etc
-if [[ "$#" -lt 3 ]]; then
-    echo "not enough parameters specified. Expected biscuit_vcf, output_cpgs.bed.gz, output_cpgs.bedgrap"
+if [[ "$#" -lt 4 ]]; then
+    echo "not enough parameters specified. Expected biscuit_vcf, reference_fasta, output_cpgs.bed.gz, output_cpgs.bedgrap"
     exit 1
 fi
 
-perl /usr/bin/biscuitVcfToBed.pl "$1" |  tee >(/bin/gzip >"$2") | cut -f 1-4 | sort -k1,1 -k2,2n -S 12G | /usr/bin/perl -ne 'print $_ if $_ =~ /^(chr)?[1-9]?[0-9|X|Y]\s/' >"$3"
+/usr/bin/biscuit vcf2bed -t cg -k 1 -e "$1" | /usr/bin/biscuit mergecg "$2.gz" /dev/stdin -k 2 |  tee >(/bin/gzip >"$3") | cut -f 1-4 | sort -k1,1 -k2,2n -S 12G | /usr/bin/perl -ne 'print $_ if $_ =~ /^(chr)?[1-9]?[0-9|X|Y]\s/' >"$4"
```

Comment:
I kind of wish we could make the `.gz` a secondaryFile for the reference for this pipeline... but that doesn't really work with the `string` version.  It's a bit of an interesting that that this script expects to be passed most of a filename for `$2`.

Reply:
Added an error with a little more info

Reply:
removed this completely - the reference file was jacked up, which caused the error.  the whole gz thing is unnecessary